doc/htmldoc/model_details/hh_details.rst

```diff
@@ -0,0 +1,67 @@
+.. _hh_details:
+
+A note on Hodgkin Huxley models
+===============================
+
+Asynchronicity between spikes and firing pattern
+------------------------------------------------
+
+Original question:
+~~~~~~~~~~~~~~~~~~
+
+I made a Hodgkin Huxley neuron model and I can get an action potential from it, 
+but it seems there is a problem with spikes.
+Action potential shape shows one spike, but the spike recorder shows several spikes 
+and there is not any synchronicity between them.
+
+
+Answer:
+~~~~~~~
+
+This is a non-trivial topic that requires the modeler to make a choice of what they want.
+The original Hodgkin-Huxley model is not a model of a neuron, but a model for spike propagation down the squid axon.
+In this model, there is no such thing as a singular spike in the sense we commonly use it when discussing
+integrate-and-fire point neuron models.
+
+In the HH model, under certain conditions, we have a rapidly rising `Na` current depolarizing the membrane up to around +30 mV,
```

Comment:
"sodium current" or ``:math:`\text{Na}^+` ``; same for K+

Reply:
I suggest to write out "sodium" and "potassium", respectively.

Reply:
done